command/genesis/polybft_params.go

```diff
@@ -615,3 +626,16 @@ func stringSliceToAddressSlice(addrs []string) []types.Address {
 
 	return res
 }
+
+func deployProxyContracts(inputs []types.Address) []*contractInfo {
+	res := make([]*contractInfo, len(inputs))
+
+	for i, proxyAddress := range inputs {
+		res[i] = &contractInfo{
+			artifact: contractsapi.GenesisProxy,
+			address:  proxyAddress,
+		}
+	}
+
+	return res
+}
```

Comment:
I'd consider renaming the function (for example):

```suggestion
func getProxyContractsInfo(addrs []types.Address) []*contractInfo {
	res := make([]*contractInfo, len(addrs))

	for i, proxyAddress := range addrs {
		res[i] = &contractInfo{
			artifact: contractsapi.GenesisProxy,
			address:  proxyAddress,
		}
	}

	return res
}
```

Reply:
fix 8dfce2f